packages/stencil/src/components/datacom-radio/datacom-radio.css

```diff
@@ -0,0 +1,178 @@
+  color: var(--dc-primary-text-color);
+}
+.dc-radio-size-standard + label {
+  font-size: 16px;
+}
+.dc-radio-size-small + label {
+  font-size: 14px;
+}
+.dc-radio-radios:hover {
+  border: 1px solid var(--dc-primary-colour);
+  border-radius: 12px;
+}
+.dc-radio-radios:focus {
+  outline: 3px solid var(--dc-primary-colour);
+  border-radius: 12px;
+  border: 1px solid var(--dc-greyscale-05);
+}
+.dc-radio-radios:checked {
+  border: 3px solid var(--dc-background-colour);
+  background: var(--dc-primary-colour);
+  box-shadow: 0 0 0 1px var(--dc-primary-colour);
+}
+
+.dc-radio-radios:disabled {
+  background-color: #f0f0f0;
+  border: 1px solid var(--dc-greyscale-05);
+}
+.dc-radio-disabled + label {
+  color: var(--dc-greyscale-03);
+}
+.dc-radio-disabled.dc-radio-radios:checked {
+  border: 3px solid #f0f0f0;
+  background: var(--dc-greyscale-04);
+  box-shadow: 0 0 0 2px #dadada;
+}
+.dc-radio-buttons {
+  position: absolute;
+  cursor: pointer;
+  opacity: 0;
+  width: 100px;
+  height: 60px;
+  align-items: center;
+
+  overflow: hidden;
+}
+.dc-radio-buttons.dc-radio-label {
+  font-family: var(--dc-font-family);
+  font-weight: 500;
+  color: var(--dc-primary-text-color);
+  padding-left: 10px;
+  line-height: 20px;
+  text-align: center;
+}
+
+.dc-radio-buttons + label {
+  border: 2px solid #dadada;
+  display: flex;
+  /* padding: 10px 20px; */
+
+  border-radius: 8px;
+  background-color: var(--dc-background-colour);
+  font-family: var(--dc-font-family);
+  font-size: 14px;
+  font-weight: 500;
+  text-align: center;
+  align-items: center;
+  margin-left: 12px;
+  padding-left: 4px;
+}
+.dc-radio-buttons.dc-radio-size-standard + label {
+  width: 77px;
+  height: 48px;
+  font-size: 16px;
+}
+
+.dc-radio-buttons.dc-radio-size-small + label {
+  width: 87px;
+  height: 34px;
+  font-size: 14px;
+}
+
+.dc-radio-buttons:focus + label {
+  outline: 3px solid #0055ff;
+  border-radius: 12px;
+  border: 1px solid #dadada;
+}
+
+.dc-radio-buttons:hover + label {
+  background: #f0f0f0;
+  cursor: pointer;
+}
+
+.dc-radio-buttons:checked + label {
+  background-color: var(--dc-primary-colour);
+  background-image: url("data:image/svg+xml;charset=utf8,%3Csvg width='14' height='11' viewBox='0 0 14 11' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M12.922 0.373439L4.17153 9.12391L1.07793 6.03031C0.931494 5.88388 0.694057 5.88388 0.547588 6.03031L0.194057 6.38385C0.0476191 6.53028 0.0476191 6.76772 0.194057 6.91419L3.90637 10.6265C4.05281 10.7729 4.29024 10.7729 4.43671 10.6265L13.8059 1.25735C13.9523 1.11091 13.9523 0.87347 13.8059 0.727002L13.4523 0.373439C13.3059 0.227001 13.0684 0.227001 12.922 0.373439Z' fill='white'/%3E%3C/svg%3E");
+  background-size: 23px;
+  background-position: left;
+  background-repeat: no-repeat;
+  margin: 1px;
+  line-height: 12px;
+  background-position-x: 8%;
+  color: var(--dc-greyscale-05);
+  text-align: center;
+  border: 2px solid var(--dc-greyscale-05);
+  padding-left: 50px !important;
```

Comment:
Not knowing the code, but normally an "!important" is a sign of improper cascading. Leave it if there is no other way of course.